service/version_bundles.go

```diff
@@ -0,0 +1,113 @@
+package service
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func NewVersionBundles() []versionbundle.Bundle {
```

Comment:
This could have been introduced in a separate PR.

Reply:
Ack. Yes that would have kept the PR smaller.